community/modules/database/slurm-cloudsql-federation/main.tf

```diff
@@ -17,6 +17,7 @@
 locals {
   # This label allows for billing report tracking based on module.
   labels = merge(var.labels, { ghpc_module = "slurm-cloudsql-federation", ghpc_role = "database" })
+  user_managed_replication = var.user_managed_replication
```

Comment:
```suggestion
}

locals {
  user_managed_replication = var.user_managed_replication
```

Reply:
This will avoid failing a check we perform for a specific block of code. Your code is perfectly valid.